install-deps.sh

```diff
@@ -0,0 +1 @@
+env apt install $(env cat DEPENDENCIES | env grep -v "^#")
```

Comment:
this is
- exclusive to distributions using `apt`
- [a useless use of cat](http://porkmail.org/era/unix/award.html#cat)
.. and may fail if the subshell returns more than `ARG_MAX` on your system allows